### DISCUSSION TITLE
workerChunkLoading

### DISCUSSION THREAD

**Discussion by jianxingyao** [webpack/webpack#18865] [General]
对于 workerChunkLoading  这个配置项 我的理解是 配置了后 ，  new Worker(url)  代码 url 对应的文件里面  ， 如果使用 import 动态导入   那么打包后  webpack 会把  import 处理成指定的形式，但是测试下来并没有，workerChunkLoading  对应的所有配置项打包后的代码都是相同的，希望有人能解答

**Comment by alexander-akait** [comment]
Use `new Worker("...", import.meta.url)`

**Reply by jianxingyao** [reply]
没懂您的意思，第二个参数不是对象吗

**Reply by alexander-akait** [reply]
Please use English, thank you

**Reply by jianxingyao** [reply]
I don't understand what you mean, isn't the second argument an object?

**Reply by alexander-akait** [reply]
It is `import.meta.url` - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import.meta

**Reply by jianxingyao** [reply]
What I mean by this is that the second argument of the Worker needs to pass an object, and impor.meta.url is a string

**Reply by alexander-akait** [reply]
Oh, my typo - there are examples how to use https://webpack.js.org/guides/web-workers/
